.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,75 @@
+name: Bug Report
+description: Create a report to help us improve
+labels: ["bug"]
+body:
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Have you checked if this issue already exists?
+      description: |
+        Before you submit a new issue, please take a moment to check if it hasn't already been reported. This helps us streamline our efforts and ensures that your concern is addressed efficiently.
+        
+        **How to Check for Existing Issues**
+        Search:
+        Use the GitHub search bar to look for keywords related to your issue. This includes error messages, feature names, or any relevant terms.
+
+        Issues Section:
+        Navigate to the "Issues" tab on our GitHub repository to see a list of reported problems and feature requests. Filter through open and closed issues to find similar cases.
+      options:
+        - label: "yes"
+          required: true
+  - type: input
+    id: version
+    attributes:
+      label: Molgenis EMX2 version
+      description: What version of Molgenis EMX2 are you using (see footer)?
+      placeholder: "Software version: v11.31.1. Database version: 23"
```

Comment:
I suppose there is no way to default to current version number?

Reply:
We can add a pre-filled value, though this can only be hardcoded as far as I know. I did find [this](https://github.com/orgs/community/discussions/4266) on the [feedback thread](https://github.com/orgs/community/discussions/63402) and someone there referred to [this roadmap item](https://github.com/github/roadmap/issues/277), so maybe we'll be having this option in the near future.

Reply:
ok